MediaBrowser.XbmcMetadata/Parsers/SeriesNfoSeasonParser.cs

```diff
@@ -0,0 +1,60 @@
+using System.Globalization;
+using System.Xml;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Controller.Entities.TV;
+using MediaBrowser.Controller.Library;
+using MediaBrowser.Controller.Providers;
+using Microsoft.Extensions.Logging;
+
+namespace MediaBrowser.XbmcMetadata.Parsers
+{
+    /// <summary>
+    /// NFO parser for seasons based on series NFO.
+    /// </summary>
+    public class SeriesNfoSeasonParser : BaseNfoParser<Season>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SeriesNfoSeasonParser"/> class.
+        /// </summary>
+        /// <param name="logger">Instance of the <see cref="ILogger"/> interface.</param>
+        /// <param name="config">Instance of the <see cref="IConfigurationManager"/> interface.</param>
+        /// <param name="providerManager">Instance of the <see cref="IProviderManager"/> interface.</param>
+        /// <param name="userManager">Instance of the <see cref="IUserManager"/> interface.</param>
+        /// <param name="userDataManager">Instance of the <see cref="IUserDataManager"/> interface.</param>
+        /// <param name="directoryService">Instance of the <see cref="IDirectoryService"/> interface.</param>
+        public SeriesNfoSeasonParser(
+            ILogger logger,
+            IConfigurationManager config,
+            IProviderManager providerManager,
+            IUserManager userManager,
+            IUserDataManager userDataManager,
+            IDirectoryService directoryService)
```

Comment:
I really dislike the NfoParsers having so many dependencies. They are doing way too much.

Reply:
Mainly caused by the base NFO parser - but we should be able to slim it down (not here ofc).